### PR TITLE
[DO NOT MERGE] - Aligning Docker version between containers/image and skopeo

### PR DESCRIPTION
**Pull request by hferentschik**
See also https://github.com/containers/image/pull/328

### CODE REVIEW

File: vendor.conf.tmp [hunk 1]
```diff
@@ -0,0 +1,46 @@
+github.com/urfave/cli v1.17.0
```
Comment (mtrmac) [inline vendor.conf.tmp]: Please drop this file.

File: vendor.conf [hunk 2]
```diff
@@ -1,23 +1,25 @@
 github.com/urfave/cli v1.17.0
-github.com/containers/image master
+github.com/containers/image issue-327 https://github.com/hferentschik/image.git
```
Comment (runcom) [inline vendor.conf]: can you vendor this from `master` again?

File: vendor.conf [hunk 3]
```diff
@@ -1,23 +1,25 @@
...
 github.com/opencontainers/go-digest master
 gopkg.in/cheggaaa/pb.v1 ad4efe000aa550bb54918c06ebbadc0ff17687b9 https://github.com/cheggaaa/pb
 github.com/containers/storage master
 github.com/sirupsen/logrus v1.0.0
+github.com/Sirupsen/logrus v1.0.0
```
Comment (runcom) [inline vendor.conf]: mmm which of these is correct? why do we have both?

File: vendor.conf (hunk 2 above)
Comment (hferentschik) [reply vendor.conf]: Sorry, that was my intention

File: vendor.conf (hunk 3 above)
Comment (hferentschik) [reply vendor.conf]: github.com/sirupsen/logrus v1.0.0 is the correct one now. There was a major issue around logrus when it got renamed from github.com/Sirupsen/logrus to github.com/sirupsen/logrus.

**Comment by hferentschik** [comment]
@runcom pushed an update

File: vendor.conf (hunk 3 above)
Comment (runcom) [reply vendor.conf]: right, my question was more "should we keep both?!"

**Comment by runcom** [comment]
> @runcom pushed an update

looks ok locally, will merge after Travis is done testing. Thanks!

**Comment by hferentschik** [comment]
Thanks

File: vendor.conf [hunk 4]
```diff
@@ -27,7 +28,7 @@ github.com/opencontainers/runc master
 github.com/opencontainers/image-spec v1.0.0
 # -- start OCI image validation requirements.
 github.com/opencontainers/runtime-spec v1.0.0
-github.com/opencontainers/image-tools 6d941547fa1df31900990b3fb47ec2468c9c6469
+github.com/opencontainers/image-tools da84dc9dddc823a32f543e60323f841d12429c51
```
Comment (mtrmac) [inline vendor.conf]: This reverts image-tools to an older commit, notably using upper-case `Sirupsen`.  But this commit does not actually update the vendored package?!

File: vendor.conf [hunk 5]
```diff
@@ -48,4 +49,3 @@ github.com/opencontainers/selinux master
 golang.org/x/sys master
 github.com/tchap/go-patricia v2.2.6
 github.com/BurntSushi/toml master
-github.com/pquerna/ffjson d49c2bc1aa135aad0c6f4fc2056623ec78f5d5ac
```
Comment (mtrmac) [inline vendor.conf]: And this removed item also does not correspond to removing any files from `vendor`?!

**Comment by runcom** [comment]
@mtrmac thanks for spotting that

**Pull request merged**